functions/set-daily-thought.js

```diff
@@ -1,6 +1,68 @@
+require('dotenv').config()
```

Comment:
```suggestion
require('dotenv').config();
```
Really need to setup some linter